.circleci/config.yml

```diff
@@ -61,6 +61,26 @@ jobs:
           fingerprints:
             - "02:df:a5:6a:53:9a:f5:5d:bd:a6:fc:b2:db:9b:c9:47" # disable-secrets-detection
             - "f5:25:6a:e5:ac:4b:84:fb:60:54:14:82:f1:e9:6c:f9" # disable-secrets-detection
+      - run:
+          name: Check if Contributor branch
+          when: always
+          command: |
+            echo $CONTRIB_BRANCH
+            if [ -z $CONTRIB_BRANCH ]
+            then
+              echo "Skipping, contributor branch not given."
+              exit 0
+            else
+              USER=$(echo $CONTRIB_BRANCH | cut -d ":" -f 1)
+              BRANCH=$(echo $CONTRIB_BRANCH | cut -d ":" -f 2)
+
+              echo 'Copy the changes from the contributor branch'
+              git remote add $USER git@github.com:$USER/content.git
+              git fetch $USER $BRANCH
+              git checkout -t $USER/$BRANCH
```

Comment:
better to take his changes i think, for instance take a list of packs

Reply:
done